package.json

```diff
@@ -63,10 +61,9 @@
     "jest-cli": "^18.1.0",
     "jsdom": "^7.0.2",
     "mocha": "^3.2.0",
-    "moment": ">=2.16.0",
     "pre-commit": "^1.1.3",
-    "react": ">=0.13",
-    "react-addons-test-utils": ">=0.13",
+    "react": ">=15 <16",
```

Comment:
Technically previous version still matches that but since react versioning has been change I just updated it. I can reverse that but still it would be good to add upper limit `<16` in case there are changes that would break the library.